File: Python/Easy/two_sum.py
class Solution:
    def two_sum(self, nums: list[int], target: int) -> list[int]:
        lastIndex = len(nums)
        for i in range(lastIndex):
            for j in range(i+1, lastIndex):
                if nums[j] == target - nums[i]:
                    return [i, j]
        return []
    
solution = Solution()
answer = solution.two_sum([-1,8,19,4,7,-5,3,9], 11)
print(answer)

File: typescript/Easy/two_sum.ts
// * Create class
class Solution {
    // ? Create two_sum method
  two_sum(nums: number[], target: number) {
    // Loop indexes to get every element of the list
    for (let i = 0; i < nums.length; i++) {
        // Loop again but to get a next element of the list
      for (let j = i + 1; j < nums.length; j++) {
        // Check --> target = nums[i] + nums[j]
        if (nums[j] === target - nums[i]) {
            // return right answer as list which includes indexes
          return [i, j];
        }
      }
    }
    // if no answer, just return empty list
    return [];
  }
}

/* 
Now, We can test our solution model
*/
const nums: number[] = [1, 22, -4, 6, 9, 11, 5, 4];
const target: number = 23;

const solution = new Solution();
const answer = solution.two_sum(nums, target);
console.log(answer);

// =========== Saidkdodirov Tolibjon ==============